graphql/schema.graphql

```diff
@@ -67,6 +67,7 @@ type Query {
   instanceTypes: [String!]!
   distroTaskQueue(distroId: String!): [TaskQueueItem!]!
   taskQueueDistros: [TaskQueueDistro!]!
+  SearchReturnInfo(taskId: String!, execution: String!): SearchReturnInfo!
```

Comment:
does it make a technical difference to graphql if the query is uppercase?

Reply:
copypasta -- it should be lowercase. Thanks!